### DISCUSSION TITLE
ElastAlert Command reported

### DISCUSSION THREAD

**Discussion by trb143** [Security-Onion-Solutions/securityonion#8057] [Unsupported Versions] [answered]
Using the latest version of SO, but any recent version will do running a Custom Alert triggers fine but when using the Command alerted there are issues as it is not possible to access programs and files outside the container.

Any ideas on what is the right way to do this.  The Alter process is fine it is just that I am not to call external utilities.

**Comment by weslambert** [comment, marked as answer]
You'll need to ensure the external commands/binaries/scripts are accessible to the container and have the appropriate permissions to be called. You should be able to drop most things into the pre-existing Elastalert directory.
